scripts/valhalla_build_config

```diff
@@ -229,6 +229,14 @@ config = {
       'max_best_paths': 4,
       'max_best_paths_shape': 100
     },
+    'bikeshare': {
+      'max_distance': 250000.0,
+      'max_locations': 50,
+      'max_matrix_distance': 200000.0,
+      'max_matrix_locations': 50,
+      'min_transit_walking_distance': 1,
+      'max_transit_walking_distance': 10000
+    },
```

Comment:
@xlqian we need to add help text for each of these below. if you look further down in this file they keys are the same but instead of the values there is a string describing what the values would mean in terms of limitations. you can basically copy paste a lot of these from other service limits which are similar.